### ISSUE TITLE
Beer song: add a test to check the whole song

### ISSUE THREAD

**Issue by rchavarria** [exercism/xecmascript#172]
Add a test that checks the whole song is returned:

```
99 bottles of beer on the wall, 99 bottles of beer.
Take one down and pass it around, 98 bottles of beer on the wall.
...
No more bottles of beer on the wall, no more bottles of beer.
Go to the store and buy some more, 99 bottles of beer on the wall.
```

**Comment by matthewmorgan** [comment]
Is @StevenACoffman going to put in a PR for this, or did you intend to do it?

**Comment by rchavarria** [comment]
I'd let @StevenACoffman decide if he wants to work on this or not. 

For the moment, I'm not planning to work on this. If @StevenACoffman isn't working on this, you can do it.

**Comment by StevenACoffman** [comment]
I'm not going to be able to get to this for a few days, so by all means
feel free. Thanks.
-Steve

**Comment by matthewmorgan** [comment]
Ok if find some time I will assign this to myself.  Otherwise it's all yours Steve.

**Comment by matthewmorgan** [comment]
Closed by #173
